<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Painter</class>
 <widget class="QMainWindow" name="Painter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>120</width>
      <height>270</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticallayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shapes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>Point</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Any</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pen Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Colors:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Choose Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="brushCheckBox">
       <property name="text">
        <string>Use Brush</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionChooseColor"/>
    <addaction name="actionHideSettings"/>
    <addaction name="actionStitchPhotos"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChooseColor">
   <property name="text">
    <string>Choose Color</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHideSettings">
   <property name="text">
    <string>Hide Settings</string>
   </property>
  </action>
  <action name="actionStitchPhotos">
   <property name="text">
    <string>Stitch Photos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
